src/animation.js

```diff
@@ -9,18 +9,21 @@ var
 	platform = require('./platform'),
 	utils = require('./utils');
 
-var ms = Math.round(1000/60);
-var prefix = ['webkit', 'moz', 'ms', 'o', ''];
-var r = 'requestAnimationFrame';
-var c = 'cancel' + utils.cap(r);
+var ms = Math.round(1000/60),
+	prefix = ['', 'webkit', 'moz', 'ms', 'o'],
+	rAF = 'requestAnimationFrame',
+	cRAF = 'cancel' + utils.cap(rAF),
```

Comment:
I realize this was existing code but it's pretty silly to add a function call so we don't have to type requestAnimationFrame again ...